infrastructure/README

```diff
@@ -0,0 +1,34 @@
+# Infrastructure
+
+This folder current manages the buckets responsible for holding uploaded lesson videos.
+
+The terraform project is broken out into multiple workspaces
+
+- production
+- staging
+- demo
+
+Each workspace should have an associated `<env>.tfvars` file. The `example_vars` can serve as a template.
+i.e. `staging.tfvars`
```

Comment:
tfvars files are excluded from the repo so you wouldn't use a tfvars file. When implemented in TF Cloud the vars are strored there.